projects/packages/forms/src/blocks/input-range/editor.scss

```diff
@@ -0,0 +1,95 @@
+.jetpack-input-range-row {
+	display: flex;
+	align-items: center;
+	width: 100%;
+	gap: 12px;
+	margin-top: 30px;
+}
+
+.jetpack-input-range__min-label,
+.jetpack-input-range__max-label {
+	font-size: .9em;
```

Comment:
Would be good to opt-into typography settings for slider and re-use default theme font size, yet allow customization.

Reply:
There are multiple instances of text in the slider designs:

<img width="616" height="215" alt="slider-text" src="https://github.com/user-attachments/assets/f6b6453e-41b2-42d9-a357-791413fa0b5d" />

A question is if we opt the slider field into typography settings, should those settings apply to all? It probably makes sense to apply them to the numeric values and labels that appear on the left/right of the input. The small value indicator on top is trickier, because it's inversed with background and text.